src/modules/ekf_att_pos_estimator/estimator_22states.cpp

```diff
@@ -2599,7 +2599,7 @@ void AttPosEKF::CovarianceInit()
     P[13][13] = sq(0.2f*dtIMU);
 
     //Wind velocities
-    P[14][14] = 0.0f;
+    P[14][14] = 0.01f;
```

Comment:
We got the value P = 0.01 for the from flight data. This is where the variance approximately seems to settle in our flights. We thought it would be a more reasonable initialisation, than P = 0, which would imply full certainty of the wind estimates at filter startup and reset.